wavefront/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: wavefront
 description: Wavefront Collector for Kubernetes
 appVersion: 1.7.5
-version: 1.7.10
+version: 1.7.11
```

Comment:
I don't think we want to make this change The open source version is currently version: 1.7.10 and the openshift version is 1.7.11